src/data/nat/modeq.lean

```diff
@@ -389,3 +389,96 @@ lemma rotate_one_eq_self_iff_eq_repeat [nonempty α] {l : list α} :
     λ h, rotate_eq_self_iff_eq_repeat.mpr h 1⟩
 
 end list
+
+--------------------------------------------------------------------------------------------------
+--------------------------------------------------------------------------------------------------
+--  PR THIS vvvv  --------------------------------------------------------------------------------
+--------------------------------------------------------------------------------------------------
+--------------------------------------------------------------------------------------------------
+
+namespace nat
+
+variables {a b c d m n : ℕ}
+
+-- Apostol, Theorem 5.3
+lemma mul_left_iff (c : ℕ) (hc : 0 < c) : a ≡ b [MOD m] ↔ c * a ≡ c * b [MOD c * m] :=
+begin
+  have hc' : (c:ℤ) ≠ 0 := by simp [hc.ne.symm],
+  simp only [modeq_iff_dvd, int.coe_nat_mul, ←mul_sub, mul_dvd_mul_iff_left hc'],
+end
+
+lemma mul_right_iff (hc : 0 < c) : a ≡ b [MOD m] ↔ a * c ≡ b * c [MOD m * c] :=
+begin
+  have hc' : (c:ℤ) ≠ 0 := by simp [hc.ne.symm],
+  simp only [modeq_iff_dvd, int.coe_nat_mul, ←mul_sub_right_distrib, mul_dvd_mul_iff_right hc'],
+end
+
+
+
+-- Apostol, Theorem 5.5
+
+lemma Apostol_5_5' (h : a ≡ b [MOD m]) (hdm : d ∣ m) :
```

Comment:
Naming results after books is very unhelpful, not to mention that mathlib already has quite a strict naming convention. Perhaps `modeq_iff_of_modeq_of_dvd` might be a more descriptive name?

Reply:
Sorry, I wasn't expecting this to be reviewed yet and I was intending to make changes (including renaming the lemmas) before switching it from "Draft" to "Ready for review".  But thanks very much for all your helpful comments!

Reply:
Ah, sorry! Should have paid don't more attention. Don't take my conments too seriously for now.

Reply:
Ok, no problem :)  Thanks for taking the time to look over it.